tests/test-images/Dockerfile.1

```diff
@@ -1,4 +1,11 @@
-FROM scratch
-MAINTAINER "William Temple <wtemple at redhat dot com>
+FROM centos
+MAINTAINER "Sally O'Malley <somalley at redhat dot com>
+ENV container docker
 
 LABEL "Name"="atomic-test-1"
+
+LABEL INSTALL "docker run --rm --spc \$SUDO_UID:\$SUDO_GID --privileged \$OPT1 FOO1 \$OPT2 FOO2 -v /:/host -e HOST=/host -e LOGDIR=${LOGDIR} -e CONFDIR=${CONFDIR} -e \${DATADIR} --name NAME IMAGE  echo I am the install label /bin/install.sh"
+LABEL RUN "docker run -t -i --rm --spc \$SUDO_UID:\$SUDO_GID \$OPT1 FOOBAR1 \$OPT2 FOOBAR2 \$OPT3 FOOBAR3 -v LOGDIR:/var/log -v DATADIR:/var/lib --name NAME IMAGE echo I am the run label run.sh"
+
+EXPOSE 80
```

Comment:
I would drop this line, as well as the --spc and --privileged options in the run and install labels.